src/deliver/render_message.rs

```diff
@@ -132,6 +152,8 @@ impl MessageRenderer {
     }
 }
 
+
+
```

Comment:
let's remove the new lines that you added here, line 110, lines 65-66. 41-42 and all other